.cargo/config.toml

```diff
@@ -0,0 +1,6 @@
+[alias]
+zig_el8 = "zigbuild --config=.cargo/zigbuild_el8.toml --target x86_64-unknown-linux-gnu.2.28"
```

Comment:
Why?

Reply:
When I was testing I needed a way to build the binary locally and upload to trevis node for testing.

Reply:
Not something I did but I guess it simplifies local dev and remote testing, do you want it gone ?

Reply:
I've added this to the base patch and will remove from this one.